src/attribution/onLCP.ts

```diff
@@ -54,21 +54,21 @@ const attributeLCP = (metric: LCPMetric): LCPMetricWithAttribution => {
               activationStart
           : 0,
       );
-      const lcpResponseEnd = Math.max(
-        lcpRequestStart,
-        lcpResourceEntry ? lcpResourceEntry.responseEnd - activationStart : 0,
-      );
-      const lcpRenderTime = Math.max(
-        lcpResponseEnd,
+      const lcpResponseEnd = Math.min(
+        // Cap at LCP time (videos continue downloading after LCP for example)
         lcpEntry.startTime - activationStart,
```

Comment:
Should this be `metric.value` as well?

Reply:
Makes sense. Updated in https://github.com/GoogleChrome/web-vitals/pull/527/commits/0136440a98826bf4081bc401b78fa8ebb3348cc0